test/addons/symlinked-module/test.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+const common = require('../../common');
+const fs = require('fs');
+const path = require('path');
+const assert = require('assert');
+
+common.refreshTmpDir();
+
+var destDir = path.resolve(common.tmpDir);
+
+var location = [path.join(destDir, 'first'), path.join(destDir, 'second')];
+fs.mkdirSync(location[0]);
+try {
+  fs.symlinkSync(location[0], location[1]);
+} catch (EPERM) {
```

Comment:
Shouldn't this be:
``js
} catch (e) {
  if (e.code !== 'EPERM') throw e;
  // ...
}
```